src/components/common/ColonyHome/ColonyFundingWidget/TokenBalanceItem.tsx

```diff
@@ -2,35 +2,28 @@ import React from 'react';
 
 import Numeral from '~shared/Numeral';
 import IconTooltip from '~shared/IconTooltip';
-// import { TokenBalancesForDomainsQuery } from '~data/index';
 import { getTokenDecimalsWithFallback } from '~utils/tokens';
 
-import { Token } from '~types';
+import { Token, Address } from '~types';
 
 import styles from './ColonyFundingWidget.css';
 
 interface Props {
-  // currentDomainId: number;
   token: Token;
   isTokenNative?: boolean;
   isNativeTokenLocked?: boolean;
+  balance?: Address;
 }
 
 const displayName = 'common.ColonyHome.ColonyFundingWidget.TokenBalanceItem';
 
 const TokenBalanceItem = ({
-  // currentDomainId,
   token: { decimals, symbol },
   isTokenNative,
   isNativeTokenLocked,
-}: Props) => {
-  // const domainBalance = balances.find(
-  //   ({ domainId }) => domainId === currentDomainId,
-  // );
-  // const balance = domainBalance && domainBalance.amount;
-  const balance = 0;
-
-  return typeof balance === 'undefined' ? null : (
+  balance,
+}: Props) =>
+  balance ? (
```

Comment:
Since it doesn't make sense to render this component without `balance` defined, my sense is that this should be rendered as 

`balance && <TokenBalanceItem balance={balance} />`, in the parent. Then you can remove the ternary in here